.editorconfig

```diff
@@ -0,0 +1,4 @@
+# 2 space indentation
+[*.java]
```

Comment:
Don't we have more than just java files? e.g. pom.xml.  It is worthwhile going through the different files and have the appropriate settings.